qbft/round_change.go

```diff
@@ -30,7 +30,7 @@ func (i *Instance) uponRoundChange(
 		roundChangeMsgContainer,
 		valCheck)
 	if err != nil {
-		return errors.Wrap(err, "could not get proposal justification for leading ronud")
```

Comment:
what does this has to do with precon-liveness?

Reply:
fixed typo

Reply:
btw,
@moshe-blox and @lior-blox 
If we later do `git revert` on this commit (the easiest way) this would be undone as well